tests/model_explainability/trustyai_service/test_trustyai_service.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from ocp_resources.namespace import Namespace
+
+from tests.model_explainability.trustyai_service.utils import wait_for_trustyai_container_terminal_state
+
+
+@pytest.mark.parametrize(
+    "model_namespace",
+    [
+        pytest.param(
+            {"name": "test-trustyai-service"},
```

Comment:
please give a meaningful name (related to what is tested) to a namespace so if there's a failure it is easy to debug